### ISSUE TITLE
Add support for maximum line length

### ISSUE THREAD

**Issue by Kernald** [hugheaves/scadformat#16]
As far as I can tell, there is some sort of support for line length (https://github.com/hugheaves/scadformat/blob/b5e8453ee74cf74edcdf172bc95f4f2f52abb12f/internal/formatter/tokenformatter.go#L118), but it's always set to max int. Having a way to wrap long lines is a blocker for me - function calls with long argument lists can result in very long lines.

**Comment by hugheaves** [comment]
Yes, there is some code to support wrapping lines, but it's incomplete. The main issue is that it doesn't properly handle comments that exceed the max line length.
